packages/langchain/src/openai/util.ts

```diff
@@ -286,3 +290,30 @@ function removeUndefinedProperties<T extends object>(obj: T): T {
   }
   return result;
 }
+
+// Workaround to support deprecated `functions` property
+// since its type `AzureOpenAiChatCompletionFunctions` contains an optional `parameters` property.
+// TODO: Remove this workaround once the `functions` property is removed from the SDK.
```

Comment:
[suggestion] Add a reminder to the next Orchestration client generation BLI, to remember to clean up this type and typeguard.

Reply:
We don't really have one right? And since this is breaking change for orchestration as well, we probably have to wait for orchestration v2 and our v2.